### ISSUE TITLE
Making any changes causes main base to be destroyed

### ISSUE THREAD

**Issue by znbaboy** [magicbear/palworld-server-toolkit#38]
[Level.zip](https://github.com/magicbear/palworld-server-toolkit/files/14234783/Level.zip)

Making any changes to the Level.sav file destorys our first base but not entirely in a weird way. Base still shows on map but is greyed out like a undiscovered fast travel point. Although, there is no Palbox present

Level.sav attached.

Just want to remove some old characters

**Comment by znbaboy** [comment]
![image](https://github.com/magicbear/palworld-server-toolkit/assets/52028136/75827ecd-fcaf-4fbc-84ff-12a1d81d44b7)

here is a screenshot of the tool loading my .sav

The top "Zurk" is my level 45 and the bottom one is my level 0. For some reason, it shows I last logged into the level 0 but it was actually the level 45 i logged into and always log into. It seems the game has my level saved on one character and my pals on another or something. After messing around trying to get this working, the best I could do is get my level 45 with all my pals lost and our main base destroyed. I cannot even destroy the main palbox and rebuild our base. Attempting to do so from the map doesn't do anything.

Any help is greatly appreciated

**Comment by znbaboy** [comment]
update, ive been messing with the tool a lot. 

the previous screenshot and explanation was slightly incorrect. My level 45 is f770fb and my level 0 is 1ef96dd. so the correct one was being logged into. However, my pals are still linked to my level 0. If I delete that character, I lose all my pals including my main base. 

I am now just trying to keep my pals and have already moved everything out of my base so I can rebuild it.

I also had to delete all the bases in game manually. If just deleted the characters without the bases, it would leave them in a state where they could not be removed.

**Comment by znbaboy** [comment]
Final update, I am 90% sure I got this working now (thanks to @magicbear also for their continuous updates and direct communication through Discord). Just waiting on friends to confirm their accounts are good after migration, but mine worked perfectly.

On version 0.7.4, I was able to migrate my saves with no issues. The buildings now migrate as part of the process and seem to keep buildings from both saves. I now have one character left that our main base is linked with. I would like to delete it, but it will delete our main base also. So for now, I have renamed it and put it in another guild:

![image](https://github.com/magicbear/palworld-server-toolkit/assets/52028136/75f71d26-968a-4dc3-978c-30ae58b9958c)

I have posed the idea to the dev to add building migration (or any other asset/container) to another player before deleting. This should allow items/buildings to be kept when deleting. It would be a cool feature to have.

Thanks again @magicbear
